paraatm/tests/aviationr_example.py

```diff
@@ -0,0 +1,50 @@
+import os
+import numpy as np
+import pandas as pd
+from paraatm.simulation_method.aviationr import AviationRisk
+import matplotlib.pyplot as plt
+import torch
+cur_dir = os.path.dirname(os.path.abspath(__file__))
+data_dir = os.path.join(cur_dir, '..', 'sample_data/')
+
+cfg = {'fp_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_GateToGate.trx',  # flight plan file
+       'mfl_file': data_dir + 'aviationR/data/TRX_DEMO_SFO_PHX_mfl.trx',  # mfl file
+       'data_file': data_dir + 'aviationR/data/', # required data for accident simulation and risk estimation
+       'model_file': data_dir + 'aviationR/model/', # Pre-trained model
+       'sim_time': 12000}  # total simulation time
+
+# call
+sim = AviationRisk(cfg)
+
+
+device = torch.device('cpu')
+if torch.cuda.is_available():
+    device = torch.device('cuda')
```

Comment:
It looks like you can remove this now that you have default handling for `simulation(device=None)`.  You should be able to call `sim.simulation()` and it will do the same thing.  And then you could also remove `import torch` from this file as well.

Reply:
Got it.